knsk.sh

```diff
@@ -97,6 +98,13 @@
         TIME=$1
         shift
       ;;
+      --kubeconfig)
+        shift
+        # Check if the port is a number
```

Comment:
Hey @mwei0210,

Awesome, it is really a usefull option! Could you just, please, fix this comment too?

Thank your for helping make this script better. Even with the news versions of k8s, this script still be useful.

Reply:
Fixed the comment and tested working. Thanks for the nice script too!